c/src/point_one/polaris/polaris.c

```diff
@@ -327,6 +320,49 @@ int Polaris_ConnectTo(PolarisContext_t* context, const char* endpoint_url,
   return POLARIS_SUCCESS;
 }
 
+/******************************************************************************/
+int Polaris_ConnectWithoutAuth(PolarisContext_t* context,
```

Comment:
I think the unique ID should be optional. Although it's probably always a good idea to use, it's nice to have the option where the server makes the IDs random for you